apps/AEPSampleApp/README.md

```diff
@@ -1,32 +1,66 @@
 # AEPSampleApp
 
-> Note: Commands are assuming you're in the root directory of the repository.
+## Prerequisites
 
-> Note: If you are new to React Native we suggest you follow the [React Native Getting Started](<https://facebook.github.io/react-native/docs/getting-started.html>) page before continuing.
+ * If you are new to React Native we suggest you follow the [React Native Setting up development environment](<https://reactnative.dev/docs/environment-setup>) React Native CLI Quickstart page before continuing.
+ * iOS pod version >= 1.11.3
 
-How to run the sample app:
+## How to run the sample app:
 
 ### Add your App Id:
+
 In `ios/**/AppDelegate.mm`, find the call to `configureWithAppId` and add your app id.
 
 In `android/**/MainApplication.java`, find the call to `configureWithAppId` and add your app id.
 
-#### Install dependecies 
+> Note: App id is configured in a mobile propery in Data Collection UI.  Refer to [configure with App ID per environment](https://developer.adobe.com/client-sdks/home/base/mobile-core/configuration/#configure-with-app-id-per-environment) page for more information.
+
+### Install dependecies 
+> Note: Commands are assuming you're in the root directory of the repository.
 
 ```
-yarn bootstrap && yarn sampleapp:ios:pod:install
+yarn install && yarn sampleapp:ios:pod:install
 ```
 
-#### Run instructions for iOS:
+### Run instructions for iOS:
 
 ```
 yarn sampleapp:ios:run
 ```
+Alternatively, running from Xcode: <br>
+In terminal > run `npx react-native start` <br>
+Double click `AEPSampleApp.xcworkspace` file. <br>
+Run the app in Xcode.
```

Comment:
How about we transform these into command line steps:
Alternatively, you can run the iOS app from Xcode by executing the following command in your terminal:
```
npx react-native start
```
Then navigate to apps/AEPSampleApp/ios and double click on the `AEPSampleApp.xcworkspace` file, or from the terminal:
```
cd apps/AEPSampleApp/ios
open AEPSampleApp.xcworkspace
```